docs/developers/intro.md

```diff
@@ -13,6 +13,8 @@ The interactive course with quizzes will guide you through everything you need t
 
 If you have questions about anything related to NEAR, you're always welcome to ask our community on [StackOverflow](https://stackoverflow.com/questions/tagged/nearprotocol) and Discord ([NEAR Discord](https://near.chat/) and [NEAR University Discord](https://discord.gg/k4pxafjMWA)).
 
+Before getting started, be sure to create your wallet on [mainnet](https://wallet.near.org/) or [testnet](https://wallet.testnet.near.org/).
```

Comment:
@sccheruku  Instead of mentioning on the Intro page how to create a testnet account, maybe we can just mention / link to https://welcome.near.university/developers/getting-started/react-tutorial#1-create-a-near-accountnear the beginning of React App Quickstart.

That feels more appropriate to me (both in terms of when to mention it and also where to point them).

Reply:
Sure, I made the change. 
Also added a section in the react-app-quickstart about creating a NEAR account, for folks that land on that page instead react-tutorial.